### ISSUE TITLE
"dotnet new blazor" creates a server-side-blazor project (instead of client-side) starting with preview6

### ISSUE THREAD

**Issue by springy76** [aspnet/AspNetCore#11491]
Inside of VS it works -- what is the according CLI command?

**Comment by springy76** [comment]
Sorry, the topic [Get Started](https://docs.microsoft.com/en-us/aspnet/core/blazor/get-started?view=aspnetcore-3.0&tabs=visual-studio) isn't on my daily read-schedule.

**Comment by lachlanwgordon** [comment]
I'm trying to create a client side blazor app with dotnet core preview 6 from terminal/vscode but it creates a server side app.

I've followed the steps from the Get Started page linked above.

Is there a way to do this in preview 6?

**Comment by springy76** [comment]
For me it worked after executing `dotnet new -i Microsoft.AspNetCore.Blazor.Templates::3.0.0-preview6.19307.2` once.

**Comment by lachlanwgordon** [comment]
Wonderful, thank you. 

Wow, that is quite obvious in the get started, don't know how I missed it.

**Comment by mjlant** [comment]
With the latest release, the entire process of setting up a Blazore client-side progect solution is just a mess. I spent five hours today, going down numerous different paths, each one ending in failure. There were missing templats, project created, but with multiple errors including missing references, mismatched .Net Core versions, generated code that did not know how to find http references, some Nuget package hell, referenes to other classes that were generated, not knowing withat App means and on and on...

I had started with a fresh download of VS Preview, .Net Core SDK and Blazore templates.

In two decades of using Visual Studio, I have never used the command line to create a project. In complete frustration, I found this page and decided to try the command line to create a Blaozr Client Side application. After a couple of tries as I figured out the syntax to create the solution/app, it worked and it worked perfectly. I suspect that some of the things I did to get the IDE path to work helped to set the stage for the command line approach to work.

Mabye my journey was atypical and I inadvertently did all of the wrong things, but at the mment using the IDE to create a Blazore client side app is an unhappy experience. Hopefully it will get a bit of love and soon morph into something useable.
